### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

File: lib/src/mock.dart [hunk 1]
```diff
@@ -726,7 +726,7 @@ typedef PostExpectation Expectation(x);
 /// `when` or `verify` context (a call "for real"), Mockito will respond with
 /// the stored canned response, if it can match the mock method parameters.
 ///
-/// The response generators include [thenReturn], [thenAnswer], and [thenThrow].
+/// The response generators include `thenReturn`, `thenAnswer`, and `thenThrow`.
```
Comment (srawlins) [inline lib/src/mock.dart]: The `[]` is how dartdoc links to other methods... why is this changed?

Comment (a14n) [reply lib/src/mock.dart]: There's a lint or hint :

> [dart] Only reference in scope identifiers in doc comments.

Actually all those methods are on `PostExpectation`. Would you prefer `[PostExpectation.thenThrow]` ?

Comment (srawlins) [reply lib/src/mock.dart]: Boo, we need [this feature](https://github.com/dart-lang/markdown/issues/172) in markdown, in order for it to do what I'd want. Ok you're change is good then. Thanks!
